resourcepolicies.md

```diff
@@ -304,6 +304,23 @@ Return codes:
 * 403 Forbidden - if you are not logged in with sufficient permissions. See the requirement in the GET Single Resource Policy endpoint
 * 404 Not found - if the resource policy doesn't exist (or was already deleted)
 
+#### Update eperson
+**POST /api/authz/resourcepolicies/<:id>/eperson**
```

Comment:
This should be a PUT. It is conceptually similar to what we do to change the owning collection of an item, see https://github.com/DSpace/RestContract/blob/main/items.md#owning-collection